iphone/Classes/TiMediaVideoPlayerProxy.m

```diff
@@ -91,7 +91,7 @@ - (NSString *)apiName
   return @"Ti.Media.VideoPlayer";
 }
 
-- (void)configureNotifications
+- (void)addNotificationObserver
```

Comment:
Please update "configureNotifications" to "addNotificationObserver" from other places also. 
e.g. inside "- (void)setUrl:(id)url_"  method.

Reply:
Good catch! But from what I see, the `setUrl:` selector is the only one using it. I also added the removal of the old observer, so we don't observe an URL twice. We definitely need more unit-tests around the whole API.